### DISCUSSION TITLE
KeePassXC- Thievingfox can Remotely retrieve Credentials From Password Managers And Windows Utilities

### DISCUSSION THREAD

**Discussion by sarahdutt28** [qutebrowser/qutebrowser#8184] [Q&A]
Hello All, 

We have come across this post where ThievingFox can get access to the KeePassXC.exe database and get the access to the credentials 
https://www.kitploit.com/2024/04/thievingfox-remotely-retrieving.html

Do we have a way to address this vulnerability yet ?

Thanks

**Comment by The-Compiler** [comment]
Not sure what lead you here, but nothing about this is related to qutebrowser. How you got to the conclusion of opening a security advisory against qutebrowser for this in addition to this discussion is truly beyond me.
